### ISSUE TITLE
The reconvert shortcut not working

### ISSUE THREAD

**Issue by shungo27** [google/mozc#555]
The reconvert shortcut not working.

```
DirectInput Ctrl Shift r Reconvert
Precomposition Ctrl Shift r Reconvert
```

Raspberry Pi OS Lite 64bit bullseye
Gnome 3.38.6
ibus-mozc 2.26.4220.102

**Comment by hiroyuki-komatsu** [comment]
Hi shungo27, thanks for the report.

Because the host applications also need to support the reconvert / undo features, those shortcuts may look like not working with those unsupported applications.

* Would you share the applications you tried?
* Would you check whether other IMEs (e.g. Anthy) support reconvert / undo features with those applications?

Thank you,

**Comment by shungo27** [comment]
#Would you share the applications you tried?
* Chromium (Chromium has ctrl + shift + r assigned to reload, so I set a different shortcut before trying.)
* gnome-nautilus
* password safe (Older version of "Secrets") (Only Undo worked.)

**Comment by hiroyuki-komatsu** [comment]
I checked the Recovert feature of the latest version (2.26.4695) with the following apps on Debian.

* Gedit (40.1): works well
* Firefox (91.7.0esr): works well
* Chrome (100): works well

For Firefox and Chrome, I used a simple textarea as follows. You can put that phrase to the URL box.

```
data:text/html,<textarea>
```
![image](https://user-images.githubusercontent.com/11634377/161412261-f7922829-1567-4559-83a6-a873feacd214.png)

It might be possible that the fix for [issue#510](https://github.com/google/mozc/issues/510) also fixed this issue.
Version 2.26.4423.102 and later contain this fix.

**Comment by hiroyuki-komatsu** [comment]
I close this issue, because

* It works on several applications
* To address issues with other specific applications, the modifications to the applications (rather than Mozc) are required.

Please feel free reopen this issue or create a new issue if the latest Mozc doesn't work on the above applications (e.g. Gedit, Firefox, Chrome).
Thank you,
